src/core/column/column.service.js

```diff
@@ -1,6 +1,14 @@
+import {isFunction} from '../services/utility';
+
 export function map(columns) {
 	return columns.reduce((memo, column) => {
 		memo[column.key] = column;
 		return memo;
 	}, {});
+}
+
+export function getValue(column) {
```

Comment:
Yeah, it would be good to keep such stuff in core.
But isn't value.js related to angular-specific model? There's something like `column.$value`